ui/react-components/components/Conflicts/Conflicts.jsx

```diff
@@ -5,14 +5,15 @@ import PropTypes from "prop-types";
 import {injectIntl} from "react-intl";
 import classNames from 'classnames';
 import {button} from "./Conflicts.module.scss";
+import WithFocusLock from "../FocusLock/WithFocusLock.jsx";
```

Comment:
naming convention is useFocusLock, please change

Reply:
Done